spec/models/expense_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Expense, type: :model do
```

Comment:
- [ ] Please make sure to add test cases for the controllers as requested by the previous reviewer. 🙏